infrastructure/src/main/java/org/corfudb/infrastructure/AutoCommitService.java

```diff
@@ -146,6 +146,11 @@ public synchronized void runAutoCommit() {
                         oldCommittedTail + 1, committedTail);
                 break;
 
+            } catch (TrimmedException trimmedException) {
+                // not sure if this line useful
+                trimmed = true;
```

Comment:
This one is used by line 111

Reply:
does this mean I can remove this catch block?